FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
+    // These are be passed to reportWin()
+    "owner": "https://www.example-dsp.com"
+    "name": "campaign123",
+    // This is used for its definition of reportWin()
+    "biddingLogicURL": "https://www.example-dsp.com/bid_logic.js"
+  },
+  // Negative interest groups are described in more detail below.
+  // Only one of negativeInterestGroup or negativeInterestGroups
+  // should be specified on a given bid.
+  "negativeInterestGroup": "campaign123_negative_interest_group",
+  "negativeInterestGroups": {
+    joiningOrigin: "https://www.example-advertiser.com",
+    interestGroupNames: [
+      "example_advertiser_negative_interest_group_a",
+      "example_advertiser_negative_interest_group_b",
+    ]
+  },
+
+  // The following fields are used to prevent replay of this bid.
+  // auctionNonce is described in greater detail below.
+  "auctionNonce": "12345678-90ab-cdef-fedcba09876543210",
+  // seller and topLevelSeller must match the auction's configuration.
+  "seller": "https://www.example-ssp.com",
+  "topLevelSeller": "https://www.another-ssp.com"
+}
+```
+
+Additional bids are not provided through the auction config passed to runAdAuction(), but rather through the response headers of a Fetch request, as described in section 6.3 Response Headers. However, the auction config will still have an additionalBids field, whose value will be a Promise with no value, used only to signal to the auction that the additional bids have arrived and are ready to be accepted in the auction.
+
+```
+navigator.runAdAuction({
+  // ...
+  'additionalBids': promiseFulfilledWhenTheFetchRequestCompletes,
+});
+```
+
+#### 6.1. Auction Nonce
+
+To prevent unintended replaying of additional bids, any auction config, whether top-level or component auction config, must include an auction nonce value if it includes additional bids.  The auction nonce is fetched provided like so:
+
+```
+const auctionNonce = navigator.createAuctionNonce();
+navigator.runAdAuction({
+  // ...
+  'auctionNonce':  auctionNonce,
+  'additionalBids': ...,
+});
+```
+
+The same nonce value will need to appear in the auctionNonce field of each additional bid associated with that auction config. Auctions that don't use additional bids don't need to create or provide an auction nonce.
+
+#### 6.2. Negative Targeting
+
+In online ad auctions for ad space, it’s sometimes useful to prevent showing an ad to certain audiences, a concept known as negative targeting. For example, you might not want to show a new customer advertisement to existing customers. New customer acquisition campaigns most often have this as a critical requirement.
+
+To facilitate negative targeting in Protected Audience auctions, each additional bid is allowed to identify one or more negative interest groups. If the user has been joined to any of the identified negative interest groups, the additional bid is dropped; otherwise it participates in the auction, competing alongside bids created by calls to generateBid().
```

Comment:
>each additional bid is allowed to identify one or more negative interest groups

An additional bid may specify _no_ negative targeting?

Reply:
Yes. An additional bid that specifies no negative interest groups is always accepted into the auction. I've updated the text here to clarify that.